src/commonMain/kotlin/com/meamoria/lexurgy/sc/SoundChanger.kt

```diff
@@ -46,17 +46,18 @@ class SoundChanger(
         romanize: Boolean = true,
         debug: (String) -> Unit = ::println,
     ): Map<String?, List<String>> {
-        val debugIndices = words.withIndex().filter { it.value in debugWords }.map { it.index }
+        val debugIndices = words.withIndex().filter { it.value in debugWords }.associateBy({ it.index }, { it.value })
```

Comment:
Please rename this variable; it's no longer just holding the indices!

Also, I think `associate { it.index to it.value }` is a bit clearer than `associateBy({ it.index }, { it.value })`.

Reply:
I wasn't sure how to call it... How does `indexToDebugWords` sound?

Will change the `associateBy`

Reply:
Works for me!